src/test/scala/ai/privado/languageEngine/go/tagger/sink/GoAPITaggerTest.scala

```diff
@@ -234,6 +234,7 @@ class GoAPITaggerTestCase3 extends GoTestBase {
         |		Name:     "John Doe",
         |		Age:      25,
         |		Location: "New York",
+        |   Email: "ggggg@gmail.com",
```

Comment:
Earlier, email was there and everything was working as expected before joern version `2.0.171` (https://github.com/Privado-Inc/privado-core/pull/857/files), when we upgraded the version to `2.0.178` this unit test case start failing ([here](https://github.com/Privado-Inc/privado-core/actions/runs/7085440575/job/19281720495#step:4:207)) so we removed the email part so that we do the release and handle this issue later. this is why we added back the email part and fixed the test.

Reply:
Ok got it now